rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -189,7 +197,7 @@ bool main_window::Init(bool with_cli_boot)
 	connect(m_thumb_stop, &QWinThumbnailToolButton::clicked, this, []()
 	{
 		gui_log.notice("User clicked stop button on thumbnail toolbar");
-		Emu.Stop();
+		Emu.GracefulShutdown(false, true);
```

Comment:
I don't think this should be async. users could spam the shutdown.

Reply:
Perhaps, but the strict error checking of Emulator::CallAfter's new overload renders such attempts fruitless.